samples/attested_tls/non_enc_client/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_executable(tls_non_enc_client
 	       client.cpp
 	       verify_callback.cpp
 	       verify_signer_openssl.cpp)
+add_dependencies(tls_non_enc_client tls_server_sign_enc)
```

Comment:
nit: could we realign the dependencies and add a line between them for readability reasons, please.